boards/arm/stm32/nucleo-f4x1re/configs/f401-fb/defconfig

```diff
@@ -0,0 +1,1894 @@
+#
```

Comment:
Seems to be a renamed `.config`. Please use `make savedefconfig` and upload a `defconfig` file instead.

Reply:
Thx. Now there is a file generated by make savedefconfig.